protobuf-kafkaconnect-converter/src/main/java/com/amazonaws/services/schemaregistry/kafkaconnect/protobuf/fromconnectschema/FieldBuilder.java

```diff
@@ -141,7 +158,17 @@ private static void setProto3Optional(
         fieldBuilder.setOneofIndex(descriptorProtoBuilder.getOneofDeclCount() - 1);
     }
 
-    private static String getTypeName(String typeName) {
-        return typeName.startsWith(".") ? typeName : "." + typeName;
+    private static String getSchemaSimpleName(String schemaName) {
+        String[] names = schemaName.split("\\.");
+        return names[names.length - 1];
+    }
+
+    private static boolean isParentLevel(String packageName, String schemaName) {
+        if (!schemaName.startsWith(packageName)) {
+            return false;
+        }
+        String[] names = schemaName.split(packageName)[1].split("\\.");
+        boolean isNotNested = names.length <= 2;
```

Comment:
What is the significance of these numbers `1` and `2`? Good to use variables to indicate what these are and why we are doing this.